packer/README

```diff
@@ -1,6 +1,14 @@
 # packer boxes for vagrant
 
-To create the boxes, run the following:
+## Requirements
+
+packer: https://www.packer.io/
+txtplate: https://github.com/aarondl/txtplate/releases
+
+## Creation
+
+To create one of the boxes do this
+
 ```bash
-packer build -only=<ONE OF: virtualbox-iso, qemu, vmware-iso> vagrant-box.json
+make <ONE OF: vagrant-virtualbox, vagrant-libvirt, vagrant-vmware, jenkins-slave, scf-cached>
```

Comment:
You don't actually have `make` rules for `vagrant-vmware` or `jenkins-slave`; and the last one actually has a target of `scf-cached-libvirt`, not `scf-cached`.